crates/diesel_models/src/query/merchant_connector_account.rs

```diff
@@ -227,4 +227,37 @@ impl MerchantConnectorAccount {
             .await
         }
     }
+
+    pub async fn find_by_profile_id(
```

Comment:
instead of `find` can this be `list`?

Reply:
and if this is an olap endpoint, let's add the feature gate. If this is used by oltp too then create a separate query for list which will be used by olap only.

Reply:
Currently we don't have `olap` feature flag in diesel_models